pom.xml

```diff
@@ -81,9 +80,9 @@
     </dependency>
 
     <dependency>
-      <groupId>log4j</groupId>
-      <artifactId>log4j</artifactId>
-      <version>${log4j.version}</version>
```

Comment:
Don't undo the log4j.version variable concept!

Reply:
Added back the version variable